### DISCUSSION TITLE
User, sys and IO

### DISCUSSION THREAD

**Discussion by teamhzipfs** [ericaltendorf/plotman#522] [General]
Hi, 

Just another miner of Chia here.

Would anyone please tell me what are these figures under user, sys and io in Plotman status? Surely I know what they mean literally but really what do they indicate?

<img width="594" alt="screen" src="https://user-images.githubusercontent.com/81176300/118518374-a524c980-b76a-11eb-83d3-0247e40ee40a.png">

For instance when the io says 10s I would really expect something like xxMB/s?

Cheers,
Chao

**Comment by thomann061** [comment]
#284
